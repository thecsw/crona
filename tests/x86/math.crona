main : void () {
    write "5 + 4 = ";
    write 5 + 4;
    write "\n";
    write "5 - 4 = ";
    write 5 - 4;
    write "\n";
    write "5 * 4 = ";
    write 5 * 4;
    write "\n";
    write "5 / 4 = ";
    write 5 / 4;
    write "\n";
    write "623 + 280 = ";
    write 623 + 280;
    write "\n";
    write "623 - 280 = ";
    write 623 - 280;
    write "\n";
    write "623 * 280 = ";
    write 623 * 280;
    write "\n";
    write "623 / 280 = ";
    write 623 / 280;
    write "\n";
}