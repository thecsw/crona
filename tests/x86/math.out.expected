5 + 4 = 9
5 - 4 = 1
5 * 4 = 20
5 / 4 = 1
623 + 280 = 903
623 - 280 = 343
623 * 280 = 174440
623 / 280 = 2
